corehq/apps/app_manager/xpath_validator/js/xpathValidator.js

```diff
@@ -0,0 +1,11 @@
+global.SchemeNumber = require('js-xpath/lib/schemeNumber').SchemeNumber;
```

Comment:
I feel like I'm missing something crucial here.

If I replace this line with `require('js-xpath/lib/schemeNumber');` it'll be useless because it won't get imported. Or if I use `require('schemeNumber')` and try to add it to package.json, `npm install` fails because `'schemeNumber' is not in the npm registry.`; and even if it was, that `require` wouldn't make it available as a global to xpath.js, which is how it's referenced from that file. Other than editing jison to output more node-compatible code for xpath.js, I can't think of any way to have this work other than `require`ing `schemeNumber` and making its `SchemeNumber` object a global. Does that make sense? Maybe I'm still missing something.